src/apps/ociobakelut/CMakeLists.txt

```diff
@@ -5,19 +5,28 @@ if(LCMS_FOUND AND (LCMS_VERSION VERSION_EQUAL 2.1 OR LCMS_VERSION VERSION_GREATE
     FIND_PACKAGE_MESSAGE(LCMS "Found lcms: ${LCMS_LIBRARIES}"
         "${LCMS_INCLUDE_DIR}")
 else()
+    find_package(Git)
+    if(NOT GIT_EXECUTABLE)
+        message("Git not found, could not build external LCMS as we cannot apply patch")
+    endif()
     message(STATUS "Using bundled lcms.")
     set(LCMS_VERSION 2.1)
     set(LCMS_BUNDLED TRUE)
+    if(CMAKE_TOOLCHAIN_FILE)
+        set(LCMS_CMAKE_ARGS ${LCMS_CMAKE_ARGS} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE})
+    endif()
+    set(LCMS_INSTALL_DIR ${CMAKE_BINARY_DIR}/ext/dist)
     ExternalProject_Add(LCMS
-        URL ${CMAKE_SOURCE_DIR}/ext/lcms2-${LCMS_VERSION}.tar.gz
-        BUILD_IN_SOURCE 1
-        CONFIGURE_COMMAND ./configure --prefix=${PROJECT_BINARY_DIR}/ext/dist --without-jpeg --without-tiff --without-zlib
-        BUILD_COMMAND make
-        INSTALL_COMMAND make install
+	PREFIX 		${CMAKE_BINARY_DIR}/lcms2-prefix
+        URL 		${CMAKE_SOURCE_DIR}/ext/lcms2-${LCMS_VERSION}.tar.gz
+	PATCH_COMMAND 	${GIT_EXECUTABLE} apply ${CMAKE_SOURCE_DIR}/ext/lcms2-${LCMS_VERSION}.patch
+        BINARY_DIR 	${CMAKE_BINARY_DIR}/ext/build/lcms2
+        INSTALL_DIR 	${LCMS_INSTALL_DIR}
+        CMAKE_ARGS 	${LCMS_CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${LCMS_INSTALL_DIR}
     )
-    set(LCMS_INCLUDE_DIRS ${PROJECT_BINARY_DIR}/ext/dist/include)
-    set(LCMS_LIBRARY_DIRS ${PROJECT_BINARY_DIR}/ext/dist/lib)
-    set(LCMS_LIBRARIES ${PROJECT_BINARY_DIR}/ext/dist/lib/liblcms2.a)
+    set(LCMS_INCLUDE_DIRS 	${PROJECT_BINARY_DIR}/ext/dist/include)
+    set(LCMS_LIBRARY_DIRS 	${PROJECT_BINARY_DIR}/ext/dist/lib)
+    set(LCMS_LIBRARIES 		${PROJECT_BINARY_DIR}/ext/dist/lib/lcms2${CMAKE_STATIC_LIBRARY_SUFFIX})
```

Comment:
Hey @ejerome, you're PR is looking like a good set of features to add to master branch, and appears to be working with VS2013/15.

There is one small bug though that prevents it from building properly on Linux/Mac.

Requires `${CMAKE_STATIC_LIBRARY_PREFIX}` before `lcms2`

Ideally this can be a quick fix for you, but seeing as how this PR has been sitting around for a while I may take the initiative and pull this into my fork with your commit maintained. Please let me know if you have the time to look at it.